modules/cert-manager-operator-update-channels.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-install.adoc
+
+:_content-type: CONCEPT
+[id="cert-manager-operator-update-channels_{context}"]
+= Understanding update channels of the {cert-manager-operator}
+
+Update channels are the mechanism by which you can declare the version of your {cert-manager-operator} in your cluster. The {cert-manager-operator} offers the following update channels:
+
+* `stable-v1`
+* `stable-v1.y`
+
+[id="stable-v1-channel_{context}"]
+== stable-v1 channel
+
+The `stable-v1` channel is the default and recommended channel while installing the {cert-manager-operator}. The `stable-v1` channel installs and updates the latest release version of the {cert-manager-operator}. Select the `stable-v1` channel if you want to use the latest stable release of the {cert-manager-operator}.
+
+The `stable-v1` channel offers the following update approval strategies:
+
+Automatic:: If you choose automatic updates for an installed {cert-manager-operator}, when a new version of the {cert-manager-operator} is available in the `stable-v1` channel, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention.
```

Comment:
This sentence has over 32 words. Is there a way to turn it into two sentences?

![Screenshot from 2023-09-28 14-06-07](https://github.com/openshift/openshift-docs/assets/57954076/0f2d2e0d-430a-45fa-9a59-4942f9f3b52f)